src/app/log-in/log-in.component.html

```diff
@@ -0,0 +1,62 @@
+<div class="container-fluid">
+    <div class="row">
+        <!-- Log in page -->
+        <div class="col-md-6">
+            <div class=" col-md-12 d-flex logInForm">
+                <div class="col-md-8">
+                    <div class="mt-5">
+                        <h2 class="text-center">LOGIN</h2>
+                    </div>
+                    <div class="text-center"
+                    style="color: #636364;">
+                        <small>Please enter your credintials</small>
+                    </div>
+                    <!-- <div *ngIf=""></div> -->
+                    <div *ngIf="isError" class="alert alert-danger mt-2 rounded-alert text-center fw-bold ">
+                        <small>Please enter a correct national ID/password</small>
```

Comment:
I recommend moving message content/all labels to a file, we normally use this library: https://www.npmjs.com/package/@ngx-translate/core